components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/model/CarbonOAuthAuthzRequest.java

```diff
@@ -29,21 +29,34 @@
 import org.wso2.carbon.identity.central.log.mgt.utils.LoggerUtils;
 import org.wso2.carbon.identity.oauth.common.OAuthConstants;
 import org.wso2.carbon.identity.oauth.config.OAuthServerConfiguration;
+import org.wso2.carbon.identity.oauth.par.model.OAuthParRequestWrapper;
 
+import javax.servlet.http.HttpServletRequest;
 import java.util.HashMap;
 import java.util.Map;
 
-import javax.servlet.http.HttpServletRequest;
-
 /**
  * OAuth 2 authorization request.
  */
 public class CarbonOAuthAuthzRequest extends OAuthAuthzRequest {
 
     private static final Log log = LogFactory.getLog(CarbonOAuthTokenRequest.class);
 
-    public CarbonOAuthAuthzRequest(HttpServletRequest request) throws OAuthSystemException, OAuthProblemException {
-        super(request);
+
+    public CarbonOAuthAuthzRequest(HttpServletRequest request) throws OAuthSystemException,
```

Comment:
@janakamarasena do we need to keep deprecate tag for public method or is it fine?  @ChitharaKarunasekera ?

Reply:
We haven't changed the method here, right?